cli/ops/os.rs

```diff
@@ -167,3 +171,13 @@ fn op_hostname(
   let hostname = sys_info::hostname().unwrap_or_else(|_| "".to_owned());
   Ok(JsonOp::Sync(json!(hostname)))
 }
+
+fn op_os_release(
+  state: &State,
+  _args: Value,
+  _zero_copy: Option<ZeroCopyBuf>,
+) -> Result<JsonOp, ErrBox> {
+  state.check_env()?;
+  let release = sys_info::os_release().unwrap_or_else(|_| "".to_owned());
```

Comment:
With Node.js, `os.release()` on Windows returns `"<major>.<minor>.<build>"` whereas `sys_info::os_release()` returns just `"<major>.<minor>"`.

I could PR a change to sys_info adding the build number but we seem to have pinned it to an older version because of Windows breakage in the latest version...

Reply:
nice, thank you.